lab_1_classify_by_unigrams/main.py

```diff
@@ -49,6 +66,13 @@ def create_language_profile(language: str, text: str) -> dict[str, str | dict[st
 
     In case of corrupt input arguments, None is returned
     """
+    frequencies = calculate_frequencies(tokenize(text))
+    if not (isinstance(language, str) and isinstance(frequencies, dict)):
```

Comment:
.

Reply:
I didn't understand what is the problem here, but I found `else` and deleted it